snow/engine/snowman/syncer/state_syncer.go

```diff
@@ -447,7 +465,7 @@ func (ss *stateSyncer) startup(ctx context.Context) error {
 	}
 
 	// list all beacons, to reach them for voting on frontier
-	for nodeID := range ss.StateSyncBeacons.Map() {
+	for nodeID := range ss.StateSyncBeacons.GetMap(ss.Ctx.SubnetID) {
```

Comment:
nit: replace `GetMap` with `GetValidatorIDs(subnetID ids.ID)`. It should be a bit more efficient, avoding all copies

Reply:
done